@shared/api/helpers/getDomainFromIssuer.ts

```diff
@@ -0,0 +1,51 @@
+import StellarSdk from "stellar-sdk";
+import { sendMessageToBackground } from "./extensionMessaging";
+import { SERVICE_TYPES } from "../../constants/services";
+import { NetworkDetails } from "../../constants/stellar";
+
+export const getDomainFromIssuer = async ({
```

Comment:
I looked into combining this with `getIconUrlFromIssuer`, I think it's cleaner / less convoluted separate rather than trying to squeeze them into the same method. My worry initially was performance, but I don't notice when making the call twice, and especially since they both using caching anyways

Reply:
Awesome! Yeah that's better. You're right about separating concerns